beam/io-pubsub/src/main/java/cz/o2/proxima/beam/io/pubsub/PubSubDataAccessor.java

```diff
@@ -115,7 +77,12 @@
         FlatMap.of(input)
             .using(
                 (PubsubMessage in, Collector<StreamElement> ctx) ->
-                    toElement(entity, in.getPayload()).ifPresent(ctx::collect),
+                    PubSubUtils.messageToElement(
+                            entity,
+                            UUID.randomUUID().toString(),
```

Comment:
@je-ik do you remember why PB beam IO ignore messageId? I assume it's because that value can came from different source (not proxima) and in that case is safer to use random UUID, is it right?

Maybe here we can have some config flag like `uniqueMessageId` (default false) and use it when true.

Reply:
The idea historically was to pull a common (operator independent) code to a module that would be shaded into each operator-dependent module. In case of pubsub, that would mean, we would create a `proxima-io-pubsub` module in (`root/io/pubsub`), which would then be made a dependency of `proxima-direct-io-pubsub` and `proxima-beam-io-pubsub`, ideally shaded inside. If we don't want to do that right away, we can at least unify the code.